src/codemodder/context.py

```diff
@@ -39,14 +40,16 @@ def __init__(
         dry_run: bool,
         verbose: bool,
         registry: CodemodRegistry,
-    ):
+        repo_manager: PythonRepoManager,
+    ):  # pylint: disable=too-many-arguments
         self.directory = directory
         self.dry_run = dry_run
         self.verbose = verbose
         self._results_by_codemod = {}
         self._failures_by_codemod = {}
         self.dependencies = {}
         self.registry = registry
+        self.repo_manager = repo_manager
```

Comment:
Very minor thing but we should add the type declaration at the class definition above.

Reply:
Are you referring to something else that's not already line 43, `repo_manager: PythonRepoManager,`?

Reply:
Yes, it should be added on line 35 above too.